tidy3d/components/data/monitor_data.py

```diff
@@ -326,6 +327,43 @@ def normalize(self, source_spectrum_fn: Callable[[float], complex]) -> FieldData
 
         return self.copy(update=fields_norm)
 
+    def to_source(
+        self, source_time: SourceTimeType, center: Coordinate, size: Coordinate = None
```

Comment:
size is not a coordinate but rather a `Size` type: https://github.com/flexcompute/tidy3d/blob/develop/tidy3d/components/types.py#L142